lib/RestructureCFG/InlineDivergentScopesPass.cpp

```diff
@@ -0,0 +1,390 @@
+//
+// Copyright rev.ng Labs Srl. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/CFG.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/LLVMContext.h"
+#include "llvm/Support/GenericDomTree.h"
+
+#include "revng/RestructureCFG/InlineDivergentScopesPass.h"
+#include "revng/RestructureCFG/ScopeGraphGraphTraits.h"
+#include "revng/Support/Assert.h"
+#include "revng/Support/GraphAlgorithms.h"
+#include "revng/Support/IRHelpers.h"
+
+using namespace llvm;
+
+// Debug logger
+Logger<> InlineDivergentScopesPassLogger("inline-divergent-scopes");
```

Comment:
Unused.

Reply:
Removed this here https://github.com/revng/revng/blob/a3b1cbac550057e016f37c8f9eb201e8ab290b82/lib/RestructureCFG/InlineDivergentScopesPass.cpp#L19